client/src/views/create-sport-field.js

```diff
@@ -0,0 +1,146 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import { Input, Label, Card, Button } from "../components/ui";
+import { postSportField } from "../services/sport-field";
+
+function CreateSportField() {
+  const [fields, setFields] = React.useState({
+    name: "",
+    description: "",
+    image: null,
+    price_day: "",
+    price_night: ""
+  });
+  const [error, setError] = React.useState(null);
+
+  function handleChange(e) {
+    if (e.target.name === "image") {
+      setFields({ ...fields, image: e.target.files[0] });
+    } else {
+      setFields({ ...fields, [e.target.name]: e.target.value });
+    }
+  }
+
+  async function handleSubmit(e) {
+    e.preventDefault();
+    console.log(fields);
+    const formData = new FormData();
+    Object.keys(fields).forEach(key => {
+      formData.append(key, fields[key]);
+    });
+    try {
+      const club = await postSportField(formData);
+      console.log(club);
+    } catch (error) {
+      console.log(error.message);
+    }
+  }
+
+  return (
+    <Card
+      css={{
+        maxWidth: "500px",
+        margin: "auto"
+      }}
+    >
+      <form onSubmit={handleSubmit}>
+        <div css={{ marginTop: "2em" }}>
+          <Label htmlFor="name">Name</Label>
+          <Input
+            aria-label="enter name"
+            required="required"
+            autoComplete="off"
+            id="name"
+            name="name"
+            type="text"
+            placeholder="Sports field's name"
+            value={fields.name}
+            onChange={handleChange}
+          />
+        </div>
+        <div css={{ marginTop: "2em" }}>
+          <Label htmlFor="description">Description</Label>
+          <Input
+            aria-label="enter description"
+            required="required"
```

Comment:
Could you please explain what are you trying do with this. Normally `required` doesn't have a value. Check this: https://www.geeksforgeeks.org/html-required-attribute/

Reply:
Of course, now I fix it, thanks :)

Reply:
Thanks @condef5. Take care with `english` grammar: `fixed` instead `fix`.